test/features/orNotEqualsLogic.feature

```diff
@@ -0,0 +1,64 @@
+@OrAndNotEqualsLogic
```

Comment:
I would move all these scenarios into the conditionalLogic.feature file as it covers same functionality

Reply:
done